layouts/shortcodes/sso-saml-limits-info.html

```diff
@@ -0,0 +1,4 @@
+{{- $idp := .Get "idp" | default "your Identity Provider" -}}
+<div class="note info">
+  <p><strong>Note:</strong> A single SAML application in {{ $idp }} can support multiple Pulumi organizations. This allows you to manage authentication for multiple teams from one centralized configuration.</p>
```

Comment:
I hesitate to include this here, since we don't support this for SCIM and SCIM users will first walk through the SAML step. I would probably drop this note and instead issue forewarning that we don't support multi-org SCIM yet:

> If you manage multiple Pulumi organizations and plan to enable SCIM provisioning on your SAML app integration, you must configure separate applications for each organization in {{ $idp }}. Pulumi supports only one Pulumi organization per SCIM-enabled application.

Reply:
OK. How does this look? For the SAML parts of the docs:

![image](https://github.com/user-attachments/assets/f599ca09-14cc-4348-bc7f-3d38c8fe7ffd)

then for the SCIM parts:

![image](https://github.com/user-attachments/assets/bd08edc1-a85c-49a8-8624-8f77264338bd)

or were you saying just use the following for all places just like you wrote above:
> If you manage multiple Pulumi organizations and plan to enable SCIM provisioning on your SAML app integration, you must configure separate applications for each organization in {{ $idp }}. Pulumi supports only one Pulumi organization per SCIM-enabled application.

Reply:
OK. I have this just being the same note each time:

![image](https://github.com/user-attachments/assets/bb799cab-74a3-44fa-95ab-65ee3b0dac15)